### ISSUE TITLE
Cli: Error initializing data directory on first use?

### ISSUE THREAD

**Issue by mnaamani** [Joystream/joystream#542]
```
MokoBook:cli mokhtar$ yarn joystream-cli api:inspect
yarn run v1.22.4
$ /Users/mokhtar/joystream/joystream/node_modules/.bin/joystream-cli api:inspect
 ›   Error: Unexpected error while trying to initialize the data directory.(/Users/mokhtar/.local/share/joystream-cli)! Permissions issue?
error Command failed with exit code 245.
```

**Comment by Lezek123** [comment]
The CLI uses the data directory provided by oclif via `Command.config.dataDir`. According to the documentation (https://oclif.io/docs/config) this should be cross-platform, but it partially relies on an environment variable called `XDG_DATA_HOME` beeing set. If that condition is not met on Mac I think it falls back to `$HOME/.local/share` by default.

There may be some other package that will give us a more reliable cross-platform data directory location, I'll look into that. (Perhaps: https://www.npmjs.com/package/appdata-path)

**Comment by mnaamani** [comment]
Fixed in #549
